2023/10/26/main.go

```diff
@@ -0,0 +1,19 @@
+// Daily Joel 2023-10-26
+//
+// Forever loops.
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	letters := []string{"J", "o", "e", "l", "\n"}
+	for {
+		if len(letters) == 0 {
+			break
+		}
+		fmt.Print(letters[0])
+		letters = letters[1:]
+	}
+}
```

Comment:
A for loop, that will break when the number of strings in slice letters is 0

We always print index 0, the first element of the slice. For each iteration of the loop we will print the first letter and then shortening the slice by removing the first element. Until there is nothing more to print.

JOEL
0123

OEL
012
... and so on, always printing 0